wasi-sdk.cmake

```diff
@@ -38,3 +38,5 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE On)
```

Comment:
i'm not sure how these cmake files are supposed to be used to build shared library. can you explain a bit?
is it intended to always build pic even if shared lib is not used? why?

Reply:
I think I added `CMAKE_POSITION_INDEPENDENT_CODE` to `LIBCXX_CMAKE_FLAGS` in the Makefile, but it wasn't working (i.e. libc++.so and libc++abi.so still contained position-dependent code), so I added the lines to wasi-sdk.cmake and wasi-sdk-pthread.cmake.  However, I just tried moving it back to Makefile, it that works fine now, so I'll push update.